app/src/main/java/kth/jjve/xfran/adapters/EventAdapter.java

```diff
@@ -20,29 +20,29 @@
 import java.util.List;
 
 import kth.jjve.xfran.R;
-import kth.jjve.xfran.models.Event;
+import kth.jjve.xfran.models.EventInApp;
 import kth.jjve.xfran.weeklycalendar.CalendarUtils;
 
-public class EventAdapter extends ArrayAdapter<Event> {
+public class EventAdapter extends ArrayAdapter<EventInApp> {
 
 
-    public EventAdapter(@NonNull Context context, List<Event> events)
+    public EventAdapter(@NonNull Context context, List<EventInApp> eventInApps)
     {
-        super(context, 0, events);
+        super(context, 0, eventInApps);
     }
 
     @NonNull
     @Override
     public View getView(int position, @Nullable View convertView, @NonNull ViewGroup parent)
     {
-        Event event = getItem(position);
+        EventInApp eventInApp = getItem(position);
 
         if (convertView == null)
             convertView = LayoutInflater.from(getContext()).inflate(R.layout.event_cell, parent, false);
 
         TextView eventCellTV = convertView.findViewById(R.id.eventCellTV);
 
-        String eventTitle = event.getName() +" "+ CalendarUtils.cleanTime(event.getStartTime()) + " - " + CalendarUtils.cleanTime(event.getEndTime());
+        String eventTitle = eventInApp.getName() +" "+ CalendarUtils.cleanTime(eventInApp.getStartTime()) + " - " + CalendarUtils.cleanTime(eventInApp.getEndTime());
```

Comment:
this is really nitpicking, but formatting wise you use no spaces after and before the + in the first two plusses, but you do in the second two plusses. I think in the rest of the code we use formatting with + surrounded by spaces. So I would do that here too :)

Reply:
Ballmer peak me didn't notice that :D